src/main/java/roomescape/infrastructure/JdbcReservationRepository.java

```diff
@@ -8,42 +8,57 @@
 import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
 import roomescape.domain.Reservation;
+import roomescape.domain.ReservationRepository;
+import roomescape.domain.ReservationTime;
 
 @Repository
-public class ReservationRepository { // reservation 테이블에 대한 CRUD를 담당
+public class JdbcReservationRepository implements ReservationRepository {
 
   private static final int NO_ROWS_AFFECTED = 0;
 
   private final JdbcTemplate jdbcTemplate;
 
-  public ReservationRepository(JdbcTemplate jdbcTemplate) {
+  public JdbcReservationRepository(JdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
   private final RowMapper<Reservation> reservationRowMapper = (resultSet, rowNum) -> new Reservation(
-      resultSet.getLong("id"),
+      resultSet.getLong("reservation_id"),
       resultSet.getString("name"),
       resultSet.getDate("date").toLocalDate(),
-      resultSet.getTime("time").toLocalTime());
+      new ReservationTime(resultSet.getLong("time_id"),
+          resultSet.getTime("time_value").toLocalTime())
+  );
 
+  @Override
   public List<Reservation> findAll() {
-    final String sql = "select id, name, date, time from reservation";
+    final String sql =
+        "select r.id as reservation_id,"
+        + " r.name,"
+        + " r.date,"
+        + "t.id as time_id,"
+        + "t.time as time_value "
+        + "from reservation as r inner join time as t "
+        + "on r.time_id = t.id";
```

Comment:
```suggestion
    final String sql =
            """
                    select r.id as reservation_id, 
                    r.name, 
                    r.date,
                    t.id as time_id, 
                    t.time as time_value 
                    from reservation as r 
                    inner join time as t 
                    on r.time_id = t.id
                    """;
```

이렇게 + 연산자 대신 텍스트블록을 사용할 수도 있을 것 같아요

Reply:
오 이런 기능이 있었군요 감사합니다